code/modules/clothing/suits/armor.dm

```diff
@@ -205,6 +205,19 @@
 
 			return PROJECTILE_CONTINUE // complete projectile permutation
 
+/obj/item/clothing/suit/armor/greatcoat/german_overcoat
+	name = "black german overcoat"
```

Comment:
Not "black german" but "Oberth Republic uniform overcoat". Also mention in description that this is from surplus.